rollbar-java/src/test/java/com/rollbar/notifier/RollbarBaseTest.java

```diff
@@ -0,0 +1,136 @@
+package com.rollbar.notifier;
+
+import static com.rollbar.notifier.config.ConfigBuilder.withAccessToken;
+
+import static org.mockito.Mockito.verify;
+
+import com.rollbar.api.payload.Payload;
+import com.rollbar.api.payload.data.Level;
+import com.rollbar.api.payload.data.Source;
+import com.rollbar.notifier.config.Config;
+import com.rollbar.notifier.telemetry.TelemetryEventTracker;
+import com.rollbar.notifier.util.BodyFactory;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+
+public class RollbarBaseTest {
```

Comment:
`RollbarBaseTest` sounds like a base class for tests, but this doesn't seem to be one. Could you find a different name for it that reflects what is being tested here?

Reply:
the problem is that I'm testing the new methods in `RollbarBase` 
🤔 `RollbarBaseNotifierTest` ?

Reply:
> I'm testing the new methods

and nothing else, so it'd be great if the name of the test class reflected that it's a collection of telemetry related test cases.